.github/workflows/maven.yml

```diff
@@ -72,8 +74,8 @@ jobs:
       - name: Build benchmark with Maven
         # Changes can break the benchmark, so compile it now to make sure it is buildable
         run: |
-          mvn -ntp -U install -DskipTests
-          mvn -ntp -U -f src/test/greengrass-nucleus-benchmark install
+          sudo mvn -ntp -U install -DskipTests
```

Comment:
why are these sudo now?

Reply:
I added this based on @shaguptashaikh change https://github.com/aws-greengrass/aws-greengrass-nucleus/pull/956/files.

Reply:
Since you re-chown it should work without running as sudo

Reply:
According to @shaguptashaikh , "The benckmark step was failing due to some permission issues as well
 and it looks like the main build step creates a bunch of artifacts that the subsequent steps need to use".

Reply:
Nothing is created except in the target directory which you're already re owning. I think you'd be OK